config/secrets.json.example

```diff
@@ -15,6 +15,12 @@
     "useLanguage": false,
     "zawgyiProb": 0.9
   },
+  "comments": {
+    "username": "Enter username",
+    "password": "Enter password",
+    "baseUrl": "Enter base URL",
+    "count": 100
```

Comment:
`count` is not very descriptive. It looks like it's only used a few places, could we be more verbose? (I'm not actually sure what it does after reading the code, since it's just sent as params to the API)

Reply:
Yes. The ambiguous name is used in an effort to mimic the CrowdTangle content service so that the comments content service can be edited by Harshil, Daniel, or anyone else who helped to write the CT content service.

I've refactored the name to `pageCount` to reflect its role in telling the API the # of comments to return per page. However, I still want to keep the nomenclature similar so as to ensure others can make edits across both content services.

Reply:
Thanks! As long as someone a year from now can look and figure out what it means. pageCount sounds reasonable.